default.xml

```diff
@@ -14,4 +14,11 @@
 	<project path="multiap/agent" name="prplMesh-agent" />
 	<project path="multiap/tools" name="prplMesh-tools" />
 
+	<!-- upstream nng -->
+	<remote name="nng" fetch="ssh://git@github.com/nanomsg" />
+	<project path="multiap/3rdparty/nng" remote="nng" name="nng" revision="90467583b7544b68483334070518e29b00ec6d81" />
```

Comment:
I wouldn't put it under the multiap/ directory. Maybe even 3rdparty is not needed (if we add it, we should possibly also put hostap under 3rdparty).

Reply:
Not sure I agree since hostap is a fork under prpl github.
But surely safeclib and nng should be outside of multiap - I think we can keep it under 3rdparty, why not?

Reply:
Since the the script to build nng and safeclib is located under `multiap/tools` directory, so I was going to put them under `multiap` directory. so is it ok to put nng and safeclib under `3rdparty` directory in top level?

Reply:
IMHO yes, 3rdparty in top level is fine. Will you also update maptools accordingly?
If you do, please see if you can change the script to get the repos paths from the manifest (repo is written in python as well), so that we can change the manifest later on without the need to change maptools

Reply:
okay